service/src/main/java/bio/terra/workspace/service/resource/referenced/flight/create/CreateReferenceResourceFlight.java

```diff
@@ -21,10 +21,7 @@ public CreateReferenceResourceFlight(FlightMap inputParameters, Object beanBag)
 
     FlightBeanBag appContext = FlightBeanBag.getFromObject(beanBag);
 
-    // Perform access verification
-    addStep(new ValidateReferenceStep(appContext), RetryRules.cloud());
```

Comment:
Can you add a comment/documentation to this step indicating that it's dead code (for now), but might be needed again soon?